backend/src/activitypub/activities/handle.ts

```diff
@@ -92,7 +92,6 @@ export async function handle(domain: string, activity: Activity, db: D1Database,
 	const getObjectAsId = makeGetObjectAsId(activity)
 	const getActorAsId = makeGetActorAsId(activity)
 
-	console.log(activity)
```

Comment:
this is useful for debugging, we should keep it.

Reply:
I've put it back with a TODO comment for it's removal, I hope that's fine? 🙂